File: Editor3D/Python/model_elements.py/scene.py
class Scene():
    def __init__(self, id: int, models: List[PoligonalModel], flashes: List[Flash], cameras: List[Camera]):
        self.id = id
        self.flashes = flashes
        if len(models) > 0:
            self.models = models
        else
            raise Exception("At least one model")
        if len(cameras) > 0:
            self.cameras = cameras
        else
            raise Exception("At least one model")

    def method_1(self, temp: Type) -> Type:
        return pass

    def method_2(self, temp_1: Type, temp_2: Type) -> Type:
        return pass

File: Editor3D/Java/src/main/java/ModelEliments/Scene.java
package ModelEliments;

import java.lang.reflect.Type;
import java.util.List;

public class Scene {
    public int id;
    public List<PoligonalModel> models;
    public List<Flash> flashes;
    public List<Camera> cameras;

    public Scene(int id, List<PoligonalModel> models, List<Flash> flashes, List<Camera> cameras) throws Exception {
        this.id = id;
        if (models.size() > 0){
            this.models = models;
        } else {
            throw new Exception("At least one model");
        }
        this.flashes = flashes;
        if (cameras.size() > 0){
            this.cameras = cameras;
        } else {
            throw new Exception("At least one camera");
        }
    }

    public Type method1(Type temp) {
        return null;
    }

    public Type method2(Type temp1, Type temp2) {
        return null;
    }
}
